### PR TITLE
Rename ActorProxy invoke to invokeMethod.

### PR DESCRIPTION
**Pull request by hepin1989**
# Description

Rename ActorProxy invoke to invokeMethod.

## Issue reference

refs:https://github.com/dapr/java-sdk/issues/418

Please reference the issue this PR will close: #_[418]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ x] Code compiles correctly
* [ x] Created/updated tests
* [ x] Extended the documentation

### CODE REVIEW

**Comment by mukundansundar** [comment]
Hi @hepin1989 thanks for this PR. Can you comment `/assign` on the issue so that it can be assigned to you?

File: docs/index-all.html [hunk 1]
```diff
@@ -1705,23 +1705,23 @@ <h2 class="title">C</h2>
 <dd>
 <div class="block">
  The metadata passing to output binding components
- 
+
```
Comment (mukundansundar) [inline docs/index-all.html:1708]: Please remove all doc changes, they will be generated and added when the SDK is released. only javadoc comments are necessary so the htlm docs can be generated.

**Review by mukundansundar — CHANGES_REQUESTED** [review]
Thanks for the contribution!

File: docs/index-all.html (hunk 1 above)
Comment (hepin1989) [reply docs/index-all.html:1708]: okay

**Review by mukundansundar — APPROVED** [review]
lgtm

**Pull request merged**